src/reducers/authReducer.test.ts

```diff
@@ -14,15 +17,33 @@ describe('The auth reducer', () => {
 
   it('returns the initial state', () => {
     expect(authReducer(undefined, {})).toEqual({
-      lastActiveAt: expect.any(Number)
+      lastActiveAt: expect.any(Number),
+      lastSessionRenew: 0,
+      sessionExpiration: 0
     });
   });
 
   it('handles updateLastActiveAt', () => {
     const mockAction = updateLastActiveAt;
 
     expect(authReducer(undefined, mockAction)).toEqual({
-      lastActiveAt: fakeTime
+      lastActiveAt: fakeTime,
+      lastSessionRenew: 0,
+      sessionExpiration: 0
+    });
+  });
+
+  it('handles updateSessionRenew', () => {
+    const initialState = {
+      lastActiveAt: fakeTime,
```

Comment:
You might want to use a separate `fakeTime` here in the test setup, otherwise you can't be sure that the `updateSessionRenew` action isn't interfering with the `lastActiveAt` timestamp.